clients/admin-ui/cypress/fixtures/systems/system.json

```diff
@@ -24,7 +24,8 @@
       "special_category_legal_basis": "Explicit Consent",
       "data_shared_with_third_parties": false,
       "third_parties": [],
-      "shared_categories": []
+      "shared_categories": [],
+      "id": "pri_ac9d4dfb-d033-4b06-bc7f-968df8d125ff"
```

Comment:
just making sure these fixtures have `id`s in their privacy declarations now, since the backend does send them now.